package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "capsulahub-ui",
+  "version": "0.0.1",
+  "description": "CapsulaJS UI component library",
+  "main": "./dist/main.js",
+  "scripts": {
+    "start": "parcel src/docs/index.html",
+    "build": "webpack --mode production --progress --display-error-details"
+  },
+  "dependencies": {
+    "clean-webpack-plugin": "^0.1.19",
+    "react": "^16.2.0",
+    "react-ace": "^6.1.4",
+    "react-click-outside": "^3.0.1",
+    "react-dom": "^16.2.0",
+    "react-json-view": "^1.19.1",
+    "typeface-montserrat": "^0.0.54"
+  },
+  "devDependencies": {
+    "babel-core": "^6.26.3",
```

Comment:
What about Babel 7?

Reply:
Do we really need support async / await or another newest features ?? just for the package with a few components?

You will get a lot of pain. I already tried to do it.